scripts/publish-website.sh

```diff
@@ -2,14 +2,16 @@
 
 set -e
 
-if [[ "$TRAVIS_BRANCH" != "master" ]]; then 
-  echo 'Not deploying the website'; 
+BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
```

Comment:
TRAVIS_PULL_REQUEST_BRANCH can also be master branch on the forked library.
I think you may need to just check `if (!TRAVIS_PULL_REQUEST && TRAVIS_BRANCH==master)`

Reply:
Ping @danielmschmidt

Reply:
Pong, will implement it somewhen this week, if you like to see it faster feel free to just push to the branch ;)